### ISSUE TITLE
Update DCGM Persistent Tool to Utilize dcgm-exporter

### ISSUE THREAD

**Issue by Maxusmusti** [distributed-system-analysis/pbench#2115]
This will allow us to stop using the old python2 script, and will also allow for both more metric options, as well as significantly simplified metric customization.

More details at:
https://github.com/NVIDIA/gpu-monitoring-tools

**Comment by Maxusmusti** [comment]
@ashishkamra This will be added either once PR #2076 is merged, or once PR #14 for issue #2122 is merged on top of #2076 (currently complete, awaiting review). This would also be done on top of the #2076 changes, and unsure which should take precedence between this issue and issue #2121.

**Comment by dagrayvid** [comment]
@Maxusmusti Can you update here when/if there is a build available for this?

**Comment by Maxusmusti** [comment]
@dagrayvid @ashishkamra The new dcgm-exporter update has been merged into PR #2076 (which itself will be merged into main branch this week). I will update again once official rpm/image builds are available, but in the meantime, unofficial builds of the new update can be found at  https://copr.fedorainfracloud.org/coprs/meyceoz/pbench-test and quay.io/meyceoz. Also the official live-metric-visualizer and prom-graf-visualizer have already been updated to reflect the change to dcgm-exporter.

**Comment by Maxusmusti** [comment]
I forgot to update again a couple weeks ago, but these changes are in the main branch now, so any builds off of the current main branch will include the update.
